app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,31 @@
  *= require foundation_and_overrides
 
  */
+
+ .profile-photo img {
+  width: 3em;
```

Comment:
Properties should be ordered background, float, margin-right, margin-top, max-height, width